coralproject/talk
# [CORL 568] more user drawer links

## What does this PR do?
- turns the "in reply to" and "moderated by" labels on moderation cards into buttons that open the user drawer for the parent user or moderator. 

## How do I test this PR?
- go to moderation view and click on "in reply to" and "moderated by". User drawer should appear.
